src/docs/guides/fastify.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deploy a Fastify App
+---
+
+Fastify is a high-performance, low-overhead web framework for Node.js, designed to deliver an exceptional developer experience.
+
+This guide covers how to deploy a Fastify app on Railway in four ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+## One-Click Deploy from a Template
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/ZZ50Bj)
+
+You can also choose from a <a href="https://railway.app/templates?q=fastify" target="_blank">variety of Fastify app templates</a> created by the community.
+
+## Deploy from a GitHub Repo
+
+To deploy a Fastify app on Railway directly from GitHub, follow the steps below:
+
+1. Fork the basic <a href="https://github.com/railwayapp-templates/fastify-hello-world" target="_blank">fastify-hello-world GitHub repo</a>.
+2. Create a <a href="https://railway.app/new" target="_blank">New Project.</a>
+3. Click **Deploy from GitHub repo**.
+4. Select your `fastify-hello-world` repo.
+    - Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.
+5. Click **Deploy Now**.
+
+Once your Fastify app is successfully deployed, a Railway service (also referred to as an app) will be created for you. By default, this service will not be publicly accessible.
+
+To set up a publicly accessible URL for your app, navigate to the **Networking** section in the **Settings** tab of your new service and click on **Generate Domain**.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727377689/docs/languages-and-frameworks/fastifyhelloworld_xbkrry.png"
+alt="screenshot of new project menu with deploy from github selected"
+layout="responsive"
+width={2447} height={1029} quality={100} />
+
+**Note:** Railway requires that Fastify's `.listen` method for the `host` be set to `0.0.0.0` or `::`. 
```

Comment:
When an app listens only on `0.0.0.0` it will only be available over the public network, if a user has their app listen on that, down the line they will not be able to talk to the service privately, perhaps only mention `::`

Reply:
Wow, Never knew that. Awesome, will update to mention only `::`.